spec/index.md

```diff
@@ -5878,7 +5884,12 @@ Q.canister_id = ic_principal
 Q.method_name = 'fetch_canister_logs'
 Q.arg = candid(A)
 A.canister_id = effective_canister_id
-S[A.canister_id].canister_log_visibility = Public or Q.sender in S[A.canister_id].controllers
+S[A.canister_id].canister_log_visibility = 
+  Public 
```

Comment:
```suggestion
(S[A.canister_id].canister_log_visibility = Public)
```
it's a top-level disjuction that we're splitting across lines

Reply:
done.